pickee_frontend/src/components/FriendsSelector/FriendsSelector.js

```diff
@@ -8,34 +8,79 @@ export default {
         ItemList
     },
     methods: {
-        removeFriend: function(targetItem) {
-            const targetItemIndex = this.selectedFriends.findIndex(sourceItem => sourceItem.id === targetItem.id);
-            this.selectedFriends.splice(targetItemIndex, 1);
+        removeFriend: function(friend) {
+            this.associatedFriends.find(ascFriend => friend.id === ascFriend.id).hidden = false;
+            const friendIndex = this.selectedFriends.findIndex(sourceItem => sourceItem.id === friend.id);
+            this.selectedFriends.splice(friendIndex, 1);
         },
-        addFriend: async function() {
+        selectFriend: function(friend) {
```

Comment:
"friends" is a bit confusing. Maybe it's a good idea to keep referencing this domain as "associated users"

Reply:
ok

Reply:
I will rename it in another PR and every place together (there are several places where I use "friend" term). ok?

Reply:
Okay, it's just a suggestion